migrations/migration.template

```diff
@@ -0,0 +1,9 @@
+import * as knex from 'knex'
+
+export function up (knex: Knex): Knex.SchemaBuilder {
+
```

Comment:
maybe we can put the invocation of createTableifNotExist here with empty callback function with some additional comments on how the `up` should be implemented?

Reply:
@eoln should @spikerheado1234 handle this? He worked on the migrations.

Reply:
it would be nice @spikerheado1234 if you could help @ramanmangla :-) 
Sorry for the late request. I haven't noticed it on time, and your PR has been already merged.